engine/src/main/java/org/terasology/world/block/BlockRegion.java

```diff
@@ -802,6 +803,38 @@ public BlockRegion addExtents(float extentX, float extentY, float extentZ, Block
         return dest;
     }
 
+    /**
+     * calculates a 1 width region that borders the provided {@link Side} of a region
+     *
+     * @param side the side of the region
+     * @param dest will hold the result
+     * @return dest
+     */
+    public BlockRegion blockFace(Side side, BlockRegion dest) {
```

Comment:
I think it should be fine as a base method. Any particular reason it needs be in the utility class.

Reply:
1. This method with `dest` should be moved to `BlockRegionc` as default method. 
2. The in-place variant `BlockRegion::blockFace(Side side)` should be added to `BlockRegion` itself.

**Question:** is `blockFace` a good/the best name for it? Would `getSide` be better (but then it would not be clear that this would modify the region if we implement (2) from above 🤔 )